SDL/Hit.cuh

```diff
@@ -91,7 +85,7 @@ namespace SDL
       return x - n * float(2.f * float(M_PI));
     }
     CUDA_HOSTDEV inline float phi(float x, float y) {
-      return phi_mpi_pi(float(M_PI) + ATan2(-y, -x));
+      return phi_mpi_pi(float(M_PI) + atan2f(y,x));
```

Comment:
is this correct?
e.g. pi/4 is (1,1)
- old : `atan2(-1,-1)+M_PI = 0.78539816`
- new : `atan2(1,1)+M_PI = 3.9269908`;  `phi_mpi_pi(3.9269908) = -2.35619`; there is a shift by π

Reply:
I'll take care of this as well.